docs/Developers/Adding_CI.rst

```diff
@@ -0,0 +1,129 @@
+Adding CI to your Fork
+======================
+
+This tutorial will help you add the CI tools, that are used in
+coala repositories to test the code, to your forked repository.
+We recommend you to add all the CI and test everything in your
+own repository before doing a PR.
+
+Before we start adding CI its important you have a GitHub account
+and know how to fork repositories. In case you don't, you should
+have a look into our `Git Tutorial <https://coala.io/gitbasics>`_.
+
+Travis CI
+---------
+
+Travis is used to confirm that the tools install and build
+properly. It also runs the tests and confirms all test cases
+pass and have 100% coverage. These are examples of travis CI
+checks used in coala and coala-bears repository:
```

Comment:
This is not easy. Then the link targets need to be unique, otherwise our rst checks fail.